WrightTools/artists/_interact.py

```diff
@@ -0,0 +1,381 @@
+            if x_proj_norm != 0:
+                x_proj_pos /= x_proj_norm
+                x_proj_neg /= x_proj_norm
+                x_proj /= x_proj_norm
+
+            y_proj_norm = max(np.nanmax(y_proj_pos), np.nanmax(-y_proj_neg))
+            if y_proj_norm != 0:
+                y_proj_pos /= y_proj_norm
+                y_proj_neg /= y_proj_norm
+                y_proj /= y_proj_norm
+
+            alpha = 0.4
+            blue = "#517799"  # start with #87C7FF and change saturation
+            red = "#994C4C"  # start with #FF7F7F and change saturation
+
+            sp_x.fill_between(xaxis.points, x_proj_pos, 0, color=red, alpha=alpha)
+            sp_x.fill_between(xaxis.points, 0, x_proj_neg, color=blue, alpha=alpha)
+            sp_x.fill_between(xaxis.points, x_proj, 0, color="k", alpha=0.3)
+
+            sp_y.fill_betweenx(yaxis.points, y_proj_pos, 0, color=red, alpha=alpha)
+            sp_y.fill_betweenx(yaxis.points, 0, y_proj_neg, color=blue, alpha=alpha)
+            sp_y.fill_betweenx(yaxis.points, y_proj, 0, color="k", alpha=0.3)
+
+        else:
+            x_proj = np.nansum(arr, axis=0)
+            y_proj = np.nansum(arr, axis=1)
+            x_proj = norm(x_proj, channel.signed)
+            y_proj = norm(y_proj, channel.signed)
+            sp_x.fill_between(xaxis.points, x_proj, 0, color="k", alpha=0.3)
+            sp_y.fill_betweenx(yaxis.points, y_proj, 0, color="k", alpha=0.3)
+
+    draw_sideplot_projections(zi)
+
+    ax0.set_xlim(xaxis.points.min(), xaxis.points.max())
+    ax0.set_ylim(yaxis.points.min(), yaxis.points.max())
+
+    if channel.signed:
+        sp_x.set_ylim(-1.1, 1.1)
+        sp_y.set_xlim(-1.1, 1.1)
+
+    def update_sideplot_slices():
+        xlim = ax0.get_xlim()
+        ylim = ax0.get_ylim()
+        x0 = current_state.xpos
+        y0 = current_state.ypos
+
+        crosshair_hline.set_data(np.array([xlim, [y0, y0]]))
+        crosshair_vline.set_data(np.array([[x0, x0], ylim]))
+
+        arr = channel[current_state.slices].squeeze()
+        if wt_kit.get_index(data.axes, xaxis) < wt_kit.get_index(data.axes, yaxis):
+            arr = arr.T.copy()
+
+        x_temp = np.abs(xaxis.points - x0)
+        x_index = np.argmin(x_temp)
+        side_plot = arr[:, x_index].copy()
+        side_plot = norm(side_plot, channel.signed)
+        line_sp_y.set_data(side_plot, yaxis.points)
+
+        y_temp = np.abs(yaxis.points - y0)
+        y_index = np.argmin(y_temp)
+        side_plot = arr[y_index].copy()
+        side_plot = norm(side_plot, channel.signed)
+        line_sp_x.set_data(xaxis.points, side_plot)
+
+    def update_local(index):
+        slices = get_slices(sliders, data.axes, verbose=verbose)
+        if verbose:
+            print(index)
+        if radio.value_selected.strip() == "global":
+            current_state.local = False
+        if radio.value_selected.strip() == "local":
+            current_state.local = True
```

Comment:
four lines can be condensed to `current_state.local = radio.value_selected.strip() == "local"`
is strip necesary? Seems like you are comparing to known strings, not user input.